pkgs/tools/text/xsv/default.nix

```diff
@@ -13,6 +13,8 @@ rustPlatform.buildRustPackage rec {
 
   cargoSha256 = "1qk5wkjm3d4dz5fldlq7rjlm602v0l04hxrbar2j6vhcz9w2r4n6";
 
+  buildInputs = stdenv.lib.optional stdenv.isDarwin pkgs.darwin.Security;
```

Comment:
It's preferred to pass Security in all-packages.nix instead of referencing nested attributes directly. See https://github.com/NixOS/nixpkgs/commit/f4927a34e764be81fbf76ec54f5d656f3c64dff1 for an example.

Reply:
Thanks, it should be fixed now. Especially nice with a template to follow, I assumed I should pass it from somewhere, but had no idea where. (Also probably would have been surprised by the need for a different attribute path?)

Reply:
The reason they are namespaced is to avoid potential conflicts, the apple frameworks have very generic names.